builtin/credential/ldap/backend.go

```diff
@@ -132,6 +132,14 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username stri
 		return nil, logical.ErrorResponse(err.Error()), nil, nil
 	}
 
+	if cfg.AnonymousGroupSearch {
+		c.Close()
+		c, err = ldapClient.DialLDAP(cfg.ConfigEntry)
```

Comment:
To correct me at https://github.com/hashicorp/vault/pull/7578#discussion_r352586953 I actually think you have to close this new connection since `c` contains a new instance of `Connection`  (e.g. `defer c.Close()`)

Reply:
I've added an extra defer to close this connection

Reply:
Thanks